app/views/partials/clients-tab-authentication.html

```diff
@@ -0,0 +1,202 @@
+      <label>Domain</label>
+      <input class="form-control" type="text" ng-model="client.clientDomain" />
+      <div class="has-error-msg">
+        <i class="glyphicon glyphicon-warning-sign"></i
+        >{{validationRequiredMsg}}
+      </div>
+    </div>
+    <div
+      class="form-group"
+      ng-class="{ 'has-error' : ngError.certFingerprint }"
+    >
+      <label><span>**</span> Client Certificate</label>
+      <select
+        class="form-control"
+        ng-options="cert.fingerprint as 'cn=' + cert.commonName + ', o=' + cert.organization for cert in certs"
+        ng-model="client.certFingerprint"
+      >
+        <option value="" label="No client certificate"></option>
+      </select>
+      <div class="has-error-msg">
+        <i class="glyphicon glyphicon-warning-sign"></i> At least one
+        authentication mechanism is required!
+      </div>
+    </div>
+  </div>
+  <div class="form-group" ng-show="authTypes.indexOf('mutual-tls-auth') == -1">
+    <p>
+      <i class="glyphicon glyphicon-info-sign"></i> Mutual TLS authentication is
+      <b>disabled</b> on OpenHIM Core.
+    </p>
+  </div>
+</div>
+
+<div>
+  <h3>Basic Auth</h3>
+  <p class="text-muted">
+    Basic Auth requires a username and password for authentication. The username
+    is the ClientID. The password can be set below.
+  </p>
+
+  <div
+    class="form-group-2columns"
+    ng-show="authTypes.indexOf('basic-auth') != -1"
+  >
+    <div
+      class="form-group"
+      style="float: left;"
+      ng-class="{ 'has-error' : ngError.password }"
+    >
+      <label ng-if="client.passwordHash" style="float: right;">
+        <i
+          tooltip-placement="top"
+          uib-tooltip="A password has been set for this client"
+          class="glyphicon glyphicon-lock"
+        ></i>
+      </label>
+
+      <label><span>**</span> Basic Auth Password</label>
+      <input
+        class="form-control"
+        placeholder="Set/change password..."
+        type="password"
+        ng-model="temp.password"
+      />
+      <div style="margin-top: 10px;" ng-show="client.passwordHash">
+        <button class="btn btn-warning btn-xs" ng-click="removeBasicAuth()">
+          <i class="glyphicon glyphicon-remove"></i>
+        </button>
+        <label> Remove Basic Auth Details</label>
+      </div>
+      <div class="has-error-msg">
+        <i class="glyphicon glyphicon-warning-sign"></i> At least one
+        authentication mechanism is required!
+      </div>
+    </div>
+    <div
+      class="form-group"
+      style="float: right;"
+      ng-class="{ 'has-error' : ngError.passwordConfirm }"
+    >
+      <label>Confirm Password</label>
+      <input
+        class="form-control"
+        placeholder="Confirm password..."
+        type="password"
+        ng-model="temp.passwordConfirm"
+      />
+      <div class="has-error-msg">
+        <i class="glyphicon glyphicon-warning-sign"></i
+        >{{validationPasswordConfirmMsg}}
```

Comment:
I think this closing bracket should be on the same line as the closing tag above